src/engine/cache/file-cache.js

```diff
@@ -210,9 +221,15 @@ class FileCache extends BaseCache {
       clearTimeout(this.archiveTimeout)
     }
 
-    const files = await fs.readdir(this.archiveFolder)
-    const referenceDate = new Date().getTime()
+    let files = []
+    try {
+      files = await fs.readdir(this.fileFolder)
+    } catch (error) {
+      this.logger.error(error)
```

Comment:
a comment explaining why this can happen and why silently ignoring it is the right way to go would be more than helpful